### DISCUSSION TITLE
Stuck on downloading OpenGFX at 200% on macOS - ensure to grant access to Documents folder

### DISCUSSION THREAD

**Discussion by michalc** [OpenTTD/OpenTTD#11334] [General]
In case this helps someone else, took me a while to figure out.

After following the instructions to compile at https://github.com/OpenTTD/OpenTTD/blob/master/COMPILING.md on macOS Ventura, running the `openttd` binary from the terminal would result in a blue screen asking me to download OpenGFX. If I agreed, it would appear to download, but then get stuck on 200%

"6.73 MiB of 3.36 MiB downloaded (200%)"

![image](https://github.com/OpenTTD/OpenTTD/assets/13877/62f9c462-83be-4bf5-b7a5-16c98565c1ec)

What solved this was granting the terminal (in my case, iTerm2) permission in System Settings -> Privacy & Security -> Files and folder access to the Documents folder. Then my compiled version would run as expected.

(Of course - thank you to everyone who has been involved in this project... it's amazing on so many levels)

**Comment by TrueBrain** [comment]
And as a PSA for you, the blue-waze issue has been resolved in our nightlies (either by switching to the "nightly" branch on Steam or downloading the latest nightly from openttd.org).
